daemon/mgr/container_logs.go

```diff
@@ -0,0 +1,138 @@
+package mgr
+
+import (
+	"context"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	"github.com/alibaba/pouch/apis/types"
+	"github.com/alibaba/pouch/daemon/logger"
+	"github.com/alibaba/pouch/daemon/logger/jsonfile"
+	"github.com/alibaba/pouch/pkg/errtypes"
+	"github.com/alibaba/pouch/pkg/utils"
+
+	pkgerrors "github.com/pkg/errors"
+)
+
+var watchTimeout = 300 * time.Millisecond
+
+// Logs is used to return log created by the container.
+func (mgr *ContainerManager) Logs(ctx context.Context, name string, logOpt *types.ContainerLogsOptions) (<-chan *logger.LogMessage, bool, error) {
+	c, err := mgr.container(name)
+	if err != nil {
+		return nil, false, err
+	}
+
+	if !(logOpt.ShowStdout || logOpt.ShowStderr) {
+		return nil, false, pkgerrors.Wrapf(errtypes.ErrInvalidParam, "you must choose at least one stream")
+	}
+
+	cfg, err := convContainerLogsOptionsToReadConfig(logOpt)
+	if err != nil {
+		return nil, false, err
+	}
+
+	// NOTE: created container doesn't create IO.
+	if c.IsCreated() {
+		msgCh := make(chan *logger.LogMessage, 1)
+		close(msgCh)
```

Comment:
I cannot quite follow the above two line of code. Why to initialize a channel and close it immediately?

Reply:
The created container doesn't have the log file in current design. 

For this case, we should fast return with closed channel. The receiver receives from closed channel and it will break the dead loop and finish the `writeLogStream`. 

I just hold the change scope in the log side and don't want to change the logic of initialize container in this commit. Does it make senses? Maybe I can add the TODO to explain why I do.